doc/nrf/ug_thread_certification.rst

```diff
@@ -63,16 +63,19 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the following overlay files:
+   * :file:`../common/overlay-no-logging.conf`
+   * :file:`../common/overlay-ble.conf`
```

Comment:
Not sure placing Kconfig fragments like this is the best approach.
It looks nice to use, but it makes it much harder for the users to know where to look for additional configurations for a given sample.

For example, I can look here:
https://github.com/nrfconnect/sdk-zephyr/tree/master/samples/net/sockets/echo_server
And I immediately know that there exists additional configuration options for the echo_server, like bluetooth, openthread, 802154, etc.

With this approach, it will be much harder for users to know where to look.
Also, are all Kconfig fragments suitable for all samples ?
Does it make sense to use `overlay-thread_1_2_ftd.conf` with the https://github.com/nrfconnect/sdk-nrf/tree/master/samples/openthread/coap_client sample ?

Reply:
As a generic note, we should try to align with other samples and applications in our sdk (and its deps), including the tree structure. I support the comment and thoughts from @tejlmand here.

Reply:
I concur with @Rallare

Reply:
Take a look at the structure now https://github.com/canisLupus1313/sdk-nrf/tree/thread_1_2_overlay_move/samples/openthread/cli